edge/pkg/edged/dockertools/docker_manager.go

```diff
@@ -40,6 +40,7 @@ import (
 	"github.com/docker/docker/api/types/network"
 	dockerapi "github.com/docker/docker/client"
 	"github.com/docker/docker/pkg/jsonmessage"
+	"github.com/kubeedge/kubeedge/edge/pkg/edged/apis"
```

Comment:
This import should be along with kubeedge imports batch

Reply:
done